cantools/subparsers/generate_c_source.py

```diff
@@ -84,4 +85,7 @@ def add_subparser(subparsers):
     generate_c_source_parser.add_argument(
         'infile',
         help='Input database file.')
+    generate_c_source_parser.add_argument(
+        '--node',
+        help='Generate pack functions only for messages from sender node and unpack functions for messages from all other nodes.')
```

Comment:
Line too long, limit to around 85 characters.

Reply:
`Generate pack functions only for the sender node and unpack functions for other nodes`
I think this should be clear enough and is now within 85 characters.

Reply:
I suggest: `Generate pack functions for messages sent from given node, and unpack functions for other messages.`